appveyor.yml

```diff
@@ -1,21 +1,24 @@
 cache:
   - node_modules
+  - "%LOCALAPPDATA%\\Yarn"
+
+init:
+  - git config --global core.autocrlf true
```

Comment:
@stof You could have been a bit more clear as to what you were talking about.
Your feedback leaves allot of room for misunderstanding.

This is not best practices considering windows.There are cases where you will run into trouble with this. Since I don't think we will have those here I'll remove `autocrlf` line endings on your request.